tests/ext/sqlalchemy_core/test_postgres.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from .test_base import connection, User, session, Base
+
+from sqlalchemy import create_engine
+from sqlalchemy.dialects.postgresql import insert as pg_insert
+
+from aws_xray_sdk.core import xray_recorder, patch
+from aws_xray_sdk.core.context import Context
+
+import testing.postgresql
+
+
+@pytest.fixture()
+def postgres_db():
+    with testing.postgresql.Postgresql() as postgresql:
+        yield postgresql
+
+
+@pytest.fixture()
+def sanitized_db_url(postgres_db):
+    dsn = postgres_db.dsn()
+    return 'postgresql://{user}@{host}:{port}/{db}'.format(
+        user=dsn['user'],
+        host=dsn['host'],
+        port=dsn['port'],
+        db=dsn['database'],
+    )
+
+
+@pytest.fixture()
+def engine(postgres_db):
```

Comment:
Would it be possible to re-use the `engine` specified in the `test_base`? Not a deal-breaker, just a good-to-have.